src/drivers/ms4525_airspeed/ms4525_airspeed.cpp

```diff
@@ -417,7 +400,7 @@ start(int i2c_bus)
 	int fd;
 
 	if (g_dev != nullptr) {
-		errx(1, "already started");
+		PX4_ERR("already started");
```

Comment:
Be aware that `errx` exited the task, so exchanging it with `PX4_ERR` is usually not enough.

Reply:
Thanks, I'll take a closer look. One of my goals for the drivers is to remove the boiler plate that's manually duplicated across nearly all of these drivers. Either a Driver ModuleBase or a shared launcher/driver runner.